.github/workflows/publish-c-apidocs.yml

```diff
@@ -21,27 +27,50 @@ jobs:
           sudo apt-get install libclang-cpp14
           wget https://www.doxygen.nl/files/doxygen-1.9.6.linux.bin.tar.gz
           tar xvzf doxygen-1.9.6.linux.bin.tar.gz
-      - name: Set commit ID
-        id: vars
-        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
       - name: Run doxygen
         run: |
           mkdir -p build/doxygen
           cd docs/c_cxx
           ../../doxygen-1.9.6/bin/doxygen
-      - uses: actions/checkout@v2
+      - name: Log source commit
+        run: git rev-parse --short HEAD > build/doxygen/html/source-version.txt
+      - name: Download deployed site
+        uses: dawidd6/action-download-artifact@v2
         with:
-          ref: gh-pages
-          clean: false
-      - name: Move API docs into target area
+          name: github-pages
+          workflow: jekyll-gh-pages.yml
```

Comment:
is this a new file? I didn't see it

Reply:
Good spot! That is the config change I mentioned in the PR description. I've updated the description to make this more explicit